app/frontend/src/components/base/BaseInternationalization.vue

```diff
@@ -78,6 +77,7 @@ export default {
           ? 'zhHant'
           : lang.keyword;
       this.setMultiLanguage(lang.keyword);
+      sessionStorage.setItem('lang', lang.keyword);
```

Comment:
This will save the lang in the sessionStorage, but when you refresh the page, the language selected will still say English.